<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierServiceDialog</class>
 <widget class="QDialog" name="ModifierServiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1504</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1504</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>14</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>1291</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 241, 244);
border: 3px solid black;
border-radius: 25px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>441</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 187, 197);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_Nom">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid transparent;
border-radius: 15px;
background-color: rgb(227, 241, 244);
</string>
     </property>
     <property name="text">
      <string>‎</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_Description">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid transparent;
border-radius: 15px;
background-color: rgb(227, 241, 244);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Description &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Service&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;</string>
     </property>
     <property name="text">
      <string>Nom Service</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>390</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border:transparent;
qproperty-iconSize: 60px 60px; /* Ajuste la taille de l'icône */</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/img/ai_artificial_intelligence_robot_chip_brain_technology_icon_179495.png</normaloff>:/img/ai_artificial_intelligence_robot_chip_brain_technology_icon_179495.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>441</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 187, 197);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>420</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>101</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt; .XC./X&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 4px solid black;
background-color: rgb(132, 194, 49);
border-radius: 25px;
font: 700 12pt &quot;Arial Rounded MT&quot;;

</string>
     </property>
     <property name="text">
      <string>enregistrer</string>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;</string>
     </property>
     <property name="text">
      <string>Date_Fin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;</string>
     </property>
     <property name="text">
      <string>Date_Debut</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cout">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>350</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid transparent;
border-radius: 15px;
background-color: rgb(227, 241, 244);
</string>
     </property>
     <property name="text">
      <string>‎</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;</string>
     </property>
     <property name="text">
      <string>Cout</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_Fin">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid transparent;
border-radius: 15px;
background-color: rgb(227, 241, 244);
</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_Debut">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid transparent;
border-radius: 15px;
background-color: rgb(227, 241, 244);
</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Frequence">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid transparent;
border-radius: 15px;
background-color: rgb(227, 241, 244);
</string>
     </property>
     <item>
      <property name="text">
       <string>annuel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hebdomadaire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mansuel</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;</string>
     </property>
     <property name="text">
      <string>Fréquence</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid transparent ; /* Ajoute une bordure */
font: 700 12pt &quot;Arial Rounded MT&quot;;</string>
     </property>
     <property name="text">
      <string>Statut</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Statut">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid transparent;
border-radius: 15px;
background-color: rgb(227, 241, 244);
</string>
     </property>
     <item>
      <property name="text">
       <string>actif</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>suspendu</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>650</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 241, 244);
font: 900 12pt &quot;Segoe UI&quot;;
border-image-source: linear-gradient(to right, black, white); /* Définir le gradient */
border-image-slice: 1; /* Permet d'appliquer le gradient sur toute la bordure */
border-radius: 10px; /* Ajuste selon le degré d'arrondi souhaité */</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
